frontend/app/ui_components/wiki-toolbar-directive.js

```diff
@@ -27,7 +27,7 @@
 //++
 
 module.exports = function() {
-  var HELP_LINK_HTML = '<a href="/help/wiki_syntax" title="' + I18n.t('js.inplace.link_formatting_help') + '" class="icon icon-help" onclick="window.open(&quot;/help/wiki_syntax&quot;, &quot;&quot;, &quot;resizable=yes, location=no, width=600, height=640, menubar=no, status=no, scrollbars=yes&quot;); return false;"><span class="hidden-for-sighted">' + I18n.t('js.inplace.link_formatting_help') + '</span></a>',
```

Comment:
I was irritated for a short moment because you removed the `hidden-for-sighted` span without replacement.

I assume that is suffices for a screenreader to have the title set correctly. Though I am wondering why the span existed in the first place.

**tl;dr:** You did not break accessibility, did you?